src/lib.rs

```diff
@@ -181,6 +181,17 @@ mod param_spec;
 pub use param_spec::ParamSpec;
 mod quark;
 pub use quark::Quark;
+#[macro_use]
+mod log;
+#[cfg(any(feature = "v2_46", feature = "dox"))]
```

Comment:
Why 2.46? This existed always or not?

Reply:
Still relevant

Reply:
https://developer.gnome.org/glib/stable/glib-Message-Logging.html#g-log-set-handler-full -> 2.46

Reply:
Ah I looked at the non-full version. Maybe we should also do the mutex trick here then to make it more useful? Same effect in the end...

Reply:
Hrm no, not same effect. You can have multiple loggers. Nevermind.